### PR TITLE
Add warped epoch flag

### PR DESCRIPTION
**Pull request by mixmasala**
Client tests need to check more frequently for a descriptor when epochtime.Period is shortened. This PR adds the build time flag WarpedEpoch to do so.

### CODE REVIEW

**Comment by david415** [comment]
this branch is now in conflict with master. please fix.

File: pki.go [hunk 1]
```diff
@@ -30,8 +30,14 @@ import (
 	"gopkg.in/op/go-logging.v1"
 )
 
-var errGetConsensusCanceled = errors.New("minclient/pki: consensus fetch canceled")
-var errConsensusNotFound = errors.New("minclient/pki: consensus not ready yet")
+var (
+	errGetConsensusCanceled = errors.New("minclient/pki: consensus fetch canceled")
+	errConsensusNotFound    = errors.New("minclient/pki: consensus not ready yet")
+	nextFetchTill           = 7 * (epochtime.Period / 8)
```
Comment (david415) [inline pki.go]: here you are changeing the value of nextFetchTill from by a factor of 7! Are you sure you want to do this. Isn't this new value really the old previous buggy value?

Comment (mixmasala) [reply pki.go]: good catch, this looks wrong.

Comment (mixmasala) [reply pki.go]: looks like it was a bad merge, fixed.

**Comment by david415** [comment]
looks good to me

**Pull request merged**